src/main/java/org/brapi/test/BrAPITestServer/factory/core/ListComponent.java

```diff
@@ -0,0 +1,55 @@
+package org.brapi.test.BrAPITestServer.factory.core;
+
+import io.swagger.model.Metadata;
+import io.swagger.model.core.BatchTypes;
+import io.swagger.model.core.ListSearchRequest;
+import io.swagger.model.core.ListSummary;
+import jakarta.validation.Valid;
+import org.brapi.test.BrAPITestServer.factory.BrAPIComponent;
+import org.brapi.test.BrAPITestServer.model.entity.SearchRequestEntity;
+import org.brapi.test.BrAPITestServer.service.core.ListService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class ListComponent implements BrAPIComponent<ListSummary, ListSearchRequest> {
+    private final ListService listService;
+
+    @Autowired
+    public ListComponent(ListService listService) {
+        this.listService = listService;
+    }
+
+    @Override
+    public List<ListSummary> findEntities(@Valid ListSearchRequest request, Metadata metadata) {
+        return listService.findLists(request, metadata);
+    }
+
+    @Override
+    public BatchTypes getBatchType() {
+        return BatchTypes.LISTS;
+    }
+
+    @Override
+    public SearchRequestEntity.SearchRequestTypes getSearchType() {
```

Comment:
this method is never used. please remove (see comment in _BrAPIComponent.java_

Reply:
done

Reply:
Marking resolved - this was removed